optimade/validator/validator.py

```diff
@@ -358,7 +358,11 @@ def _recurse_through_endpoint(self, endp: str) -> Tuple[bool, str]:
             # check support level of property
             prop_type = _impl_properties[prop]["type"]
             sortable = _impl_properties[prop]["sortable"]
-            optional = prop not in CONF.entry_properties[endp]["MUST"]
+            optional = (
+                prop in CONF.entry_schemas
```

Comment:
This doesn't make sense to me, shouldn't this be `endp in CONF.entry_schemas` or `prop in CONF.entry_schemas[endp]`?